modules/aws-emr-iam/main.tf

```diff
@@ -95,15 +363,26 @@ data "aws_iam_policy_document" "emr_service_policy" {
 }
 
 //The policy that attaches the minimal policy document
-resource "aws_iam_policy" "emr_service_policy" {
-  name   = var.emr_service_iam_policy_name
-  policy = data.aws_iam_policy_document.emr_service_policy.json
+resource "aws_iam_policy" "emr_service_policy_1" {
```

Comment:
What's the reason for having 2 separate emr service policies when you're attaching them to the same role?

Reply:
We were hitting this error 
```
Error: error creating IAM policy along-wu-test-14-spark-service-policy: LimitExceeded: Cannot exceed quota for PolicySize: 6144
        status code: 409, request id: 260e17fa-afce-4391-be76-7f83d97c8b47

```

See https://tamr.slack.com/archives/C025YQ0JVFZ/p1625056115010600?thread_ts=1625055697.009300&cid=C025YQ0JVFZ